### PR TITLE
DC: New API

### PR DESCRIPTION
**Pull request by schneidy**
Rewrote the DC bill scraper to work with the [new API](https://lims.dccouncil.us/api/help/index.html).

Should resolve issue #3341

### CODE REVIEW

File: scrapers/dc/bills.py [hunk 1]
```diff
@@ -26,546 +27,269 @@ class DCBillScraper(Scraper):
...
+                                    yes_count = no_count = other_count = 0
+                                    for leg_vote in act["voteDetails"]["votes"]:
+                                        mem_name = leg_vote["councilMember"]
+                                        if leg_vote["vote"] == "Yes":
+                                            yes_count += 1
+                                            v.yes(mem_name)
+                                        elif leg_vote["vote"] == "No":
+                                            no_count += 1
+                                            v.no(mem_name)
+                                        else:
```
Comment (jamesturk) [inline scrapers/dc/bills.py:277]: what kinds of other votes were you seeing?  we might want to categorize them now

File: scrapers/dc/bills.py [hunk 2]
```diff
@@ -26,546 +27,269 @@ class DCBillScraper(Scraper):
...
+
+                                    v.set_count("yes", yes_count)
+                                    v.set_count("no", no_count)
+                                    v.set_count("other", other_count)
+                                    yield v
 
-            bill.add_document_link(doc_type, doc_url, media_type=mimetype)
+                yield bill
 
     def date_format(self, d):
```
Comment (jamesturk) [inline scrapers/dc/bills.py:288]: is this still used?  I wasn't seeing it

Comment (schneidy) [reply scrapers/dc/bills.py]: Good catch! Removed it.

File: scrapers/dc/bills.py (hunk 1 above)
Comment (schneidy) [reply scrapers/dc/bills.py:288]: Added in absent, recused (abstain), and Present (other) as vote categories.

**Comment by jamesturk** [comment]
looking good, cc @showerst if you want to take a look too

**Comment by jamesturk** [comment]
also Dan can you add docs to the doc repo about the API key in the same style as the NY/etc. ones?

**Comment by showerst** [comment]
Are you guys getting a zillion insecureRequestWarning -s or is that just my env? If it's a problem for y'all too i can PR the code to disable that warning, DC is bad about certificates so it's a good idea to keep the verify=False

**Comment by schneidy** [comment]
> also Dan can you add docs to the doc repo about the API key in the same style as the NY/etc. ones?

**Comment by schneidy** [comment]
> also Dan can you add docs to the doc repo about the API key in the same style as the NY/etc. ones?

Adding now.

**Comment by schneidy** [comment]
> Are you guys getting a zillion insecureRequestWarning -s or is that just my env? If it's a problem for y'all too i can PR the code to disable that warning, DC is bad about certificates so it's a good idea to keep the verify=False

I'm also getting all of those warnings.

**Comment by showerst** [comment]
> I'm also getting all of those warnings.

Ok, i'll PR (back to this branch not master) a fix for that.

**Comment by showerst** [comment]
I added a separate MR for warning suppression, and grabbing the law number. Otherwise good to merge from my end!

**Comment by showerst** [comment]
Oops, link just in case it doesn't notify -- https://github.com/openstates/openstates-scrapers/pull/3344

**Pull request merged**